lib/text/native_text_displayer.js

```diff
@@ -202,14 +202,16 @@ shaka.text.NativeTextDisplayer = class {
    * @export
    */
   remove(start, end) {
-    if (!this.video_ || this.trackId_ < 0) {
+    // Do NOT return false when no track is selected or
+    // this method will be called endlessly by StreamingEngine
```

Comment:
According to spec, `false` should be only returned when displayer has been destroyed. Could you rearrange your comment to indicate this instead of describing StreamingEngine implementation?